var/spack/repos/builtin/packages/py-cdat-lite/package.py

```diff
@@ -40,3 +40,14 @@ class PyCdatLite(PythonPackage):
     depends_on("python@2.5:2.8", type=('build', 'run'))
     depends_on("py-numpy", type=('build', 'run'))
     depends_on('py-setuptools', type='build')
+
+    phases = ['install']
+
+    def install(self, spec, prefix):
+        """Install everything from build directory."""
+        install_args = self.install_args(spec, prefix)
+        # Combine all phases into a single setup.py command,
+        # otherwise extensions are rebuilt without rpath by install phase:
```

Comment:
> otherwise extensions are rebuilt without rpath by install phase:

I figured out how to prevent this:
```
$ python setup.py install --skip-build
```
I'll add this to the default install phase.